packages/react/src/client.ts

```diff
@@ -16,6 +16,9 @@ import merge from 'lodash/merge';
 
 export interface Node {
   id?: string | null;
+  revisions: (arg0: { first: number }) => {
```

Comment:
I think we might want a `WithRevisions` interface or something similar to this:

```ts
export interface Node {
  id?: string | null;
}

export interface WithRevisions {
  revisions: (arg0: { first: number }) => {
    edges?: { node?: Node }[];
  };
}

export interface RequiredQuery {
  posts: (args?: {
    where?: {
      categoryId?: number;
      categoryName?: string;
    };
  }) => unknown;
  post: (args: { id: string; idType?: PostIdType }) => (Node & WithRevisions) | null | undefined;
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  pages: (args?: any) => unknown;
  page: (args: { id: string; idType?: PageIdType }) => (Node & WithRevisions) | null | undefined;
  category: (args: {
    id: string;
    idType?: CategoryIdType;
  }) => Node | null | undefined;
  generalSettings?: unknown;
}
```

Since `category` does not have `revisions`.